### PR TITLE
comment out heartbeat, usb target

### PR DESCRIPTION
**Pull request by ronlawrence3**
* comment out heartbeat info message
* add environment for using a usb to update the maslow in pio:
  For example, 
    `pio run -t upload -e usb --upload-port /dev/tty.usbmodem2111301`

### CODE REVIEW

**Comment by ronlawrence3** [comment]
@BarbourSmith in case you don't get a notification...

**Comment by BarbourSmith** [comment]
Excellent!

**Comment by BarbourSmith** [comment]
I did not get a notification but I looked into the settings and it looks like it was my fault so hopefully I'll start getting them now 🤞

File: platformio.ini [hunk 1]
```diff
@@ -143,11 +143,16 @@ build_flags = ${common_esp32.build_flags} ${common_wifi.build_flags} ${common_bt
...
+extends = common_esp32_s3
+lib_deps = ${common.lib_deps} ${common.wifi_deps}
+build_flags = ${common_esp32.build_flags}  ${common_wifi.build_flags} -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1
+
 [env:wifi_s3]
 extends = common_esp32_s3
 lib_deps = ${common.lib_deps} ${common.wifi_deps}
 build_flags = ${common_esp32.build_flags}  ${common_wifi.build_flags} -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1
-upload_protocol = espota
+#upload_protocol = espota
```
Comment (ronlawrence3) [inline platformio.ini:155]: @BarbourSmith sorry, I didn't mean to comment this! you should put that back in, sorry!

Comment (BarbourSmith) [reply platformio.ini:155]: Great catch!

**Pull request merged**